core/src/main/java/com/scalar/db/api/TwoPhaseCommitTransaction.java

```diff
@@ -104,4 +105,17 @@ public interface TwoPhaseCommitTransaction extends TransactionCrudOperable {
    * @throws RollbackException if the operation fails
    */
   void rollback() throws RollbackException;
+
+  /**
+   * Aborts a transaction. This method is an alias of {@link #rollback()}.
+   *
+   * @throws AbortException if the operation fails
+   */
+  default void abort() throws AbortException {
```

Comment:
Added `abort()` to `TwoPhaseCommitTransaction` which is an alias of `rollback()`.